packages/fx-core/resource/package.nls.json

```diff
@@ -781,6 +781,15 @@
   "core.common.OutlookWebClientName2": "Outlook web access client id 2",
   "core.NewOfficeAddinOptionVSC.label": "Start with an Outlook add-in",
   "core.NewOfficeAddinOptionVSC.detail": "Use the Teams Toolkit to create a new Office add-in application.",
+  "core.summary.actionDescription": "Action %s%s",
+  "core.summary.lifecycleDescription": "Running lifecycle stage: %s(%s step(s) in total) The following actions will be executed %s",
+  "core.summary.lifecycleNotExecuted": "%s Lifecycle stage %s not executed.",
+  "core.summary.lifecycleSucceeded": "%s Lifecycle stage %s executed successfully.",
+  "core.summary.lifecycleFailed": "%s Lifecycle stage %s failed.",
+  "core.summary.actionNotExecuted": "%s not executed.",
+  "core.summary.actionFailed": "%s failed.",
+  "core.summary.actionSucceeded": "%s executed successfully.",
+  "core.summary.createdEnvFile": "Created environment file at",
```

Comment:
no clear subject for the action so suggest to use passive structure:

"Environment file was created at"